src/main/java/core/basesyntax/strategy/TransactionStrategy.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy;
+
+public interface TransactionStrategy {
+    Integer balanceUpdater(String fruitName, int quantity);
```

Comment:
You have to refactor your strategy, do not ignore comments
![image](https://github.com/mate-academy/jv-fruit-shop/assets/48316721/40892b68-83e5-4cc6-9bdf-8e8204284fd0)